SPCTR-3291: Fix - Image Gallery pagination loader issue (#3512)
brainstormforce/ultimate-addons-for-gutenberg: README.md

## Changelog ##

### x.x.x ###

> * Fix: Image Gallery - Resolved an issue in the masonry type wherein the loader is not hidden even after all the images are loaded.

Reviewer: ```suggestion
* Fix: Image Gallery - Resolved an issue with the masonry type wherein the loader would not be hidden after all the images have been loaded.
```

Author: Done.
